examples/jsm/physics/JoltPhysics.js

```diff
@@ -18,7 +18,7 @@ function getShape( geometry ) {
 		const sy = parameters.height !== undefined ? parameters.height / 2 : 0.5;
 		const sz = parameters.depth !== undefined ? parameters.depth / 2 : 0.5;
 
-		return new Jolt.BoxShape( new Jolt.Vec3( sx, sy, sz ), 0.05, null );
+		return new Jolt.BoxShape( new Jolt.Vec3( sx, sy, sz ), 0.05 * Math.min( sx, sy, sz ), null );
```

Comment:
@jrouwe What do you think about this solution?

Reply:
Given that the example uses boxes of 0.075, a convex radius of 0.05 is indeed too big (the debug version of Jolt will assert because it is bigger than half the size of the box). So this solution is better as it reduces the convex radius to 0.00375.

The default allowed penetration in Jolt is 0.02 (PhysicsSettings.mPenetrationSlop) which means the boxes can still sink into each other by this amount. Also, this will not help performance as the allowed penetration is bigger than the convex radius which causes a more expensive code path to trigger more often. You can update the physics settings through PhysicsSystem.SetPhysicsSettings.